FAQ.md

```diff
@@ -0,0 +1,19 @@
+# sequelize/cli [![Build Status](https://travis-ci.org/sequelize/cli.svg?branch=master)](https://travis-ci.org/sequelize/cli) [![Code Climate](https://codeclimate.com/github/sequelize/cli.png)](https://codeclimate.com/github/sequelize/cli) [![Greenkeeper badge](https://badges.greenkeeper.io/sequelize/cli.svg)](https://greenkeeper.io/)
+
+The Sequelize Command Line Interface (CLI) Frequently Asked Question
+
+How can I generate a model?
+This is a basic example for a model generation. Also you can add more attributes and datatypes as you needed.
```

Comment:
This line doesnt add much to command below. May be explain `name`, `attributes` instead